src/Infrastructure.Dapper/AdminConsole/Repositories/OrganizationRepository.cs

```diff
@@ -169,4 +169,16 @@ public async Task<IEnumerable<string>> GetOwnerEmailAddressesById(Guid organizat
             new { OrganizationId = organizationId },
             commandType: CommandType.StoredProcedure);
     }
+
+    public async Task EnableCollectionEnhancements(Guid id)
```

Comment:
You already named the parameter `organizationId` in the interface, lets keep it consistent with the other methods
```suggestion
    public async Task EnableCollectionEnhancements(Guid organizationId)
```